site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -129,348 +122,253 @@ If your extension needs to run on `file://` URLs or needs to operate in incognit
-    </tr>
-    <tr id="system.memory">
-      <td><code>"system.memory"</code></td>
-      <td>Gives access to the <a href="/docs/extensions/reference/system_memory/">chrome.system.memory</a> API.</td>
-    </tr>
-    <tr id="system.storage">
-      <td><code>"system.storage"</code></td>
-      <td>Gives access to the <a href="/docs/extensions/reference/system_storage/">chrome.system.storage</a> API.</td>
-    </tr>
-    <tr id="tabCapture">
-      <td><code>"tabCapture"</code></td>
-      <td>Gives access to the <a href="/docs/extensions/reference/tabCapture/">chrome.tabCapture</a> API.</td>
-    </tr>
-    <tr id="tabGroups">
-      <td><code>"tabGroups"</code></td>
-      <td>Gives access to the <a href="/docs/extensions/reference/tabGroups/">chrome.tabGroups</a> API.</td>
-    </tr>
-    <tr id="tabs">
-      <td><code>"tabs"</code></td>
-      <td>Gives access to privileged fields of the <a
-          href="/docs/extensions/reference/tabs#type-Tab"><code>Tab</code></a> objects used by several APIs
-        including <a href="/docs/extensions/reference/tabs">chrome.tabs</a> and <a href="/docs/extensions/reference/windows">chrome.windows</a>. In
-        many circumstances your extension will not need to declare the <code>"tabs"</code> permission to make use of
-        these APIs.</td>
-    </tr>
-    <tr id="topSites">
-      <td><code>"topSites"</code></td>
-      <td>Gives access to the <a href="/docs/extensions/reference/topSites/">chrome.topSites</a> API.</td>
-    </tr>
-    <tr id="tts">
-      <td><code>"tts"</code></td>
-      <td>Gives access to the <a href="/docs/extensions/reference/tts/">chrome.tts</a> API.</td>
-    </tr>
-    <tr id="ttsEngine">
-      <td><code>"ttsEngine"</code></td>
-      <td>Gives access to the <a href="/docs/extensions/reference/ttsEngine/">chrome.ttsEngine</a> API.</td>
-    </tr>
-    <tr id="unlimitedStorage">
-      <td><code>"unlimitedStorage"</code></td>
-      <td>Provides an unlimited quota for storing client-side data, such as databases and local storage files.
-        Without this permission, the extension is limited to 10 MB of local storage.<div
-          class="aside aside--note"><b>Note:</b> This permission applies only to Web SQL Database and application cache
+The following table lists the currently available permissions. See Permission warnings][doc-warning-table] for a list of permission warnings.
+
+`"activeTab"`
+: Gives temporary access to the active tab through a user gesture. See <a href="/docs/extensions/mv3/manifest/activeTab"><code>activeTab</code></a>
+for details.
+
+`"alarms"`
+: Gives access to the <a href="/docs/extensions/reference/alarms">chrome.alarms</a> API.
+
+`"background"`
+: Makes Chrome start up early (as soon as the user logs into their computer, before they launch Chrome), and shut down late (even after its last window is closed, until the user explicitly quits Chrome).
```

Comment:
It means background in the "run in the background" sense :)

Reply:
It can still be used by service workers in MV3 👍

Reply:
I had that question because 'background' was overloaded. Might an extension developer have the same question? Is there anything we can do to shore this up?